langchain/chains/combine_documents/base.py

```diff
@@ -36,6 +37,7 @@ def format_document(doc: Document, prompt: BasePromptTemplate) -> str:
 class BaseCombineDocumentsChain(Chain, ABC):
     """Base interface for chains combining documents."""
 
+    llm_chain: LLMChain
```

Comment:
Yes because we set the `prompt` variable like
`combine_docs_chain.llm_chain.prompt = CHAT_RETRIEVAL_QA_PROMPT` in the `validate_combine_docs_chain` function for the error handling and I found I have to put the type there to pass the linting check.

Reply:
@leferradkw let me know if any other questions! Hope to get the PR merged soon! :)